packages/react/src/Accordion/AccordionSection.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { ChevronDown } from '@amsterdam/design-system-react-icons'
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren, useContext, useId, useState } from 'react'
+import AccordionContext from './AccordionContext'
+import { getElement } from '../Heading/Heading'
+import { Icon } from '../Icon/Icon'
+
+export interface AccordionSectionProps extends HTMLAttributes<HTMLElement> {
+  label: string
+  expanded?: boolean
+}
+
+export const AccordionSection = forwardRef(
+  (
+    { label, expanded = false, children, className, ...otherProps }: PropsWithChildren<AccordionSectionProps>,
+    ref: ForwardedRef<HTMLDivElement>,
+  ) => {
+    const { headingLevel, section } = useContext(AccordionContext)
+    const [isExpanded, setIsExpanded] = useState(expanded)
+
+    const HeadingX = getElement(headingLevel)
+    const id = useId()
+    const buttonId = `button-${id}`
+    const panelId = `panel-${id}`
+
+    return (
+      <div className={clsx('amsterdam-accordion__section', className)} ref={ref} {...otherProps}>
+        <HeadingX className={'amsterdam-accordion__header'}>
+          <button
```

Comment:
Storybook a11y is showing a problem: "Element has focusable descendants", "Ensures interactive controls are not nested as they are not always announced by screen readers or can cause focus problems for assistive technologies". I've test Apple Voiceover and could not get the accordion content to focus (and read to me).

Reply:
Good catch, I accidentally had nested interactive elements. Not sure why VoiceOver can't access the accordion content, we have to look into that